codelists/scripts/nhsd_primary_care_refsets.json

```diff
@@ -1,7 +1,6 @@
 {
     "comment": "For loading of the xxxx_PCD_RefsetContext.txt files downloaded from NHSD PC Refset Portal",
     "organisation": "nhsd-primary-care-domain-refsets",
-    "sheet": "20241205_PCD_Refset_Content",
```

Comment:
Why is the superfluous? It might be better to add it into its own commit to better explain why the change is being made.